tidy3d/components/mode.py

```diff
@@ -91,6 +91,15 @@ class ModeSpec(Tidy3dBaseModel):
         "yz plane, the ``bend_axis`` is always 1 (the global z axis).",
     )
 
+    track: Literal["first", "last", "center"] = pd.Field(
```

Comment:
While we haven't done this yet, I think we should move away from `Literal[str1, str2, ...]` to using enums for things like this. Could you try adjusting the implementation to make the `TrackType` an enum? you can use this as an example:
https://pydantic-docs.helpmanual.io/usage/types/#enums-and-choices


```py
# types.py
from enum import Enum

class TrackEnum(str, Enum):
    first = 'first'
    last = 'last'
    center = 'center'
```
and then we can use `TrackEnum` in the type hints and do `TrackEnum.last` instead of eg. "last"